linera-service/src/linera.rs

```diff
@@ -1359,23 +1395,50 @@ impl Runnable for Job {
                         if let Some(fuel) = fuel {
                             pricing.fuel = fuel;
                         }
-                        if let Some(storage) = storage {
-                            pricing.storage = storage;
+                        if let Some(storage_num_reads) = storage_num_reads {
+                            pricing.storage_num_reads = storage_num_reads;
+                        }
+                        if let Some(storage_bytes_read) = storage_bytes_read {
+                            pricing.storage_bytes_read = storage_bytes_read;
+                        }
+                        if let Some(storage_bytes_written) = storage_bytes_written {
+                            pricing.storage_bytes_written = storage_bytes_written;
+                        }
+                        if let Some(maximum_bytes_read) = maximum_bytes_read {
+                            pricing.maximum_bytes_read = maximum_bytes_read;
+                        }
+                        if let Some(maximum_bytes_written) = maximum_bytes_written {
+                            pricing.maximum_bytes_written = maximum_bytes_written;
                         }
                         if let Some(messages) = messages {
                             pricing.messages = messages;
                         }
                         info!(
                             "Pricing:\n\
                             {:.2} base cost per block\n\
-                            {:.2} per unit of fuel used in executing user operations and messages\n\
+                            {:.2} per unit of fuel used in the read per operation\n\
+                            {:.2} per unit of fuel used in the read per byte\n\
+                            {:.2} per unit of fuel used in the write per byte\n\
```

Comment:
Do we call this "fuel" at all?
I think the first one should just be "per read operation", and the others "per byte read" and "per byte written"?

Reply:
Yes, you are right.